src/metabase/models/dashboard_card.clj

```diff
@@ -169,7 +169,7 @@
                                                   :dashboardcard_id id
                                                   {:order-by [[:position :asc]]})))
        (update-dashboard-card-series! dashboard-card series)))
-    (retrieve-dashboard-card id)))
+    true))
```

Comment:
We don't need this to return the updated dashboard card.

Reply:
nit: why `true` but not just `nil` tho? if this fn returns whether nothing or throws an exception. I don't expect to use this as a condition for anything.

Reply:
True, `nil` is probably more natural because it gives a hint that callers shouldn't depend on the return value.